app/controllers/PostTravelControllers.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+var db = require('../../config/sequelize');
+
+//Retrieve All Post Travel Information
+exports.getPostTravel = function (req, res) {
+    db.post_travel.findAll()
+        .then(function (postTravel) {
+            return res.jsonp(postTravel);
+        })
+        .catch(function (err) {
+            return res.render('error', {
+                error: err,
+                status: 500
+            })
+        });
+};
+
+//Retrieve All Post Travel Information By postTravelId
+exports.getPostTravelById = function (req, res, next, id) {
+    console.log('id => ' + id);
+    db.post_travel.find({
+        where: {
+            id: id
+        }
+    }).then(function (postTravel) {
+        if (!postTravel) {
+            return next(new Error('Failed to load postTravelId ' + id));
+        } else {
+            req.postTravel = postTravel;
+            return next();
+        }
+    }).catch(function (err) {
+        return next(err);
+    });
+};
+
+//Retrieve Post Travel Information By profileId
+exports.getPostTravelByProfileId = function (req, res, next) {
+    db.post_travel.find({ where: { profile_id: req.params.profileId } })
+        .then(function (postTravel) {
+            return res.jsonp(postTravel);
+        })
+        .catch(function (err) {
+            return res.render('error', {
+                error: err,
+                status: 500
+            })
+        });
+};
+
+//Show Post Travel
+exports.show = function (req, res) {
+    return res.jsonp(req.postTravel);
+};
+
+//Create Post Travel
+exports.createPostTravel = function (req, res, next) {
+    var message = null;
+    var postTravel = {
+        startDate: req.body.start_date,
+        endDate: req.body.end_date,
+        travelStatus: req.body.travel_status,
+        created_date: Date.now(),
+        updated_date: Date.now(),
+        country_id: req.body.country_id,
+        profile_id: req.body.profile_id
+    };
+
+    var postTravelSave = db.post_travel.build(postTravel);
+    req.body.post_travel_id = postTravelSave.id;
+
+    postTravelSave.save().then(function () {
+        return res.jsonp({
+            "result": "success"
+        });
+    }).catch(function (err) {
+        res.send({ status: 'Exception', message: err })
+    });
+};
+
+//Update Post Travel
+exports.updatePostTravel = function (req, res) {
+
+    // create a new variable to hold the article that was placed on the req object.
+    var postTravel = req.postTravel;
+
+    postTravel.updateAttributes({
+        startDate: req.body.start_date,
+        endDate: req.body.end_date,
+        travelStatus: req.body.travel_status,
+        created_date: Date.now(),
+        updated_date: Date.now(),
```

Comment:
```
created_date: Date.now(),
updated_date: Date.now(),
```
Not required as it will generate automatically when insert.

Reply:
Resolved.